src/multi-caching.ts

```diff
@@ -26,20 +30,28 @@ export function multiCaching<Caches extends Cache[]>(
 		data: T,
 		ttl?: Milliseconds | undefined,
 	) => {
-		await Promise.all(caches.map(async cache => cache.set(key, data, ttl)));
+		try {
+      await Promise.all(caches.map(async cache => cache.set(key, data, ttl)));
```

Comment:
@AdrienDoctolib I think we should talk about this - might be better to use allSettled